model/resource/agent.yaml

```diff
@@ -0,0 +1,11 @@
+groups:
+  - id: agent
+    prefix: agent
+    type: resource
+    brief: >
+        An agent
```

Comment:
At the risk of being verbose, could we give a sentence or two to describe how we define "agent" in this context? The docs seem to define agent as a [deployment pattern](https://opentelemetry.io/docs/collector/deployment/agent/), but I think these conventions are agnostic of a deployment model and use a different definition for the term.

I do see that this doesn't seem to appear anywhere in the readme output, so maybe there's no point.

Reply:
@evan-bradley I am not sure how much do we want to narrow down the definition of the agent to be honest. We could say "data collection agent" like OpAMP does, if that's useful.